Xamarin.Forms.ControlGallery.Android/RegistrarValidationService.cs

```diff
@@ -21,15 +21,15 @@ public bool Validate(VisualElement element, out string message)
 			if (element == null)
 				return true;
 
-			var renderer = Platform.Android.Platform.CreateRendererWithContext(element, _context);
+			//var renderer = Platform.Android.Platform.CreateRendererWithContext(element, _context);
```

Comment:
Pre-AppCompat Platform was public, now it's internal, not sure how to resolve this